doc/modules/metrics.rst

```diff
@@ -133,6 +133,20 @@ between two vectors. This kernel is defined as:
 where ``x`` and ``y`` are the input vectors. If :math:`\gamma = \sigma^{-2}`
 the kernel is known as the Gaussian kernel of variance :math:`\sigma^2`.
 
+.. _laplacian_kernel:
+
+Laplacian kernel
+----------------
+The function :func:`laplacian_kernel` is a variant on the radial basis 
+function kernel defined as:
+
+.. math::
+
+    k(x, y) = \exp( -\gamma \| x-y \|_1)
+
+where ``x`` and ``y`` are the input vectors and :math:`\|x-y\|_1` is the 
+Manhattan distance between the input vectors.
```

Comment:
I'm not sure I'm fully qualified to answer that question. In ML applied to learning molecular properties it sometimes offers superior performance but that's quite specific. Would such information (and a link to a relevant paper) be useful in the documentation?